packages/perseus/src/strings.ts

```diff
@@ -4,6 +4,7 @@ import type {ErrorCodes} from "@khanacademy/perseus-score";
  * The translated strings that are used to render Perseus.
  */
 export type PerseusStrings = {
+    characterCount: ({used, num}: {used: number; num: number}) => string;
```

Comment:
It might be useful to add a comment explaining that `num` has to be named `num` to indicate that its value controls pluralization.